### DISCUSSION TITLE
How to specify package version in config.yaml under the "Package Access" section in Verdaccio docs?

### DISCUSSION THREAD

**Discussion by jitterfunc** [verdaccio/verdaccio#4439] [Q&A]
I want to specify a package's version (or a range of versions of a package) in the config.yaml, e.g. only minor version upgrades of a package should be possible via Verdaccio and all other major versions of that package would be blocked.

**Use case:** 
I want to prevent breaking changes from getting introduced as a result of upgrading to a major version of a package and any such upgrade has to go through a proper intake approval process.

**Comment by juanpicado** [comment]
Not possible via configuration, the idea is similar to others have been already proposed in another discussion.
